src/insns/acperm_32bit.adoc

```diff
@@ -52,9 +52,9 @@ The rules from <<acperm_rules>> must be followed when removing permissions.
 |  4  (RV32 only) | <<x_perm>>   | <<r_perm>>
 |  5  (RV32 only) | <<w_perm>>   | not(<<c_perm>>) or <<lm_perm>>
 |  6  (RV32 only) | <<x_perm>>   | <<w_perm>> or <<c_perm>>
-|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>>
-|  8              | <<lm_perm>>  | <<c_perm>> and <<r_perm>>
-|  9              | <<sl_perm>>  | <<c_perm>>
+|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>> and <<lm_perm>>
+|  8              | <<lm_perm>>  | <<c_perm>> and <<r_perm>> and (<<w_perm>> or <<el_perm>>)
```

Comment:
The second part of the constraint should be RV32-only - for RV64 we should be able to have LM without EL.

Reply:
same comment as above